### PR TITLE
Logentries added to templates

### PR DESCRIPTION
**Pull request by nhaynes**
This stack uses the [official logentries docker image](https://hub.docker.com/r/logentries/docker-logentries/) used to send docker logs to their service.

### CODE REVIEW

File: templates/logentries/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+docker-logentries:
+  image: logentries/docker-logentries:0.2.1
+  command: ${logentries_command}
+  restart: always
+  volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
+  labels:
+    io.rancher.scheduler.global: "true"
+    io.rancher.container.hostname_override: container_name
```
Comment (rawmind0) [inline templates/logentries/0/docker-compose.yml]: Could you please use compose version 2 format??

**Comment by rawmind0** [comment]
Hi @nhaynes .. Thanks so much for the PR and contribute.. :)

Could you please use compose version 2 format in docker-compose.yml file??

**Comment by nhaynes** [comment]
Sorry about that, I think that covers it?

**Comment by rawmind0** [comment]
Thanks @nhaynes ... 

LGTM..

**Pull request merged**